ansible/roles/postgresql/templates/postgresql.conf.j2

```diff
@@ -295,11 +295,11 @@ logging_collector = on        # Enable capturing of stderr and csvlog
 # These are only used if logging_collector is on:
 log_directory = 'pg_log'       # directory where log files are written,
                     # can be absolute or relative to PGDATA
-log_filename = 'postgresql-%Y-%m-%d_%H%M%S.log'    # log file name pattern,
+log_filename = 'postgresql-%a.log'    # log file name pattern,
                     # can include strftime() escapes
 #log_file_mode = 0600           # creation mode for log files,
                     # begin with 0 to use octal notation
-#log_truncate_on_rotation = off     # If on, an existing log file with the
+log_truncate_on_rotation = on     # If on, an existing log file with the
```

Comment:
when this is on, it will rewrite any file. so monday's old logs will be rewritten